kerrokantasi/settings/base.py

```diff
@@ -103,15 +114,8 @@
     'TEST_REQUEST_DEFAULT_FORMAT': 'json',
 }
 
+JWT_AUTH['JWT_PAYLOAD_GET_USER_ID_HANDLER'] = 'helusers.jwt.get_user_id_from_payload_handler'
```

Comment:
Why is this separately here? It could be initialized in the `JWT_AUTH` dict with the rest...

Reply:
This is not really a configurable thing. Ie. configurables are at top and non-configurables sit beyond the veil.